tests/cypress/tests-e2e/legacyParamImport.cy.js

```diff
@@ -411,4 +411,70 @@ describe('Test on legacy param import', () => {
             cy.readStoreValue('state.position.projection.epsgNumber').should('eq', 3857)
         })
     })
+
+    context('Bod Layer Id import', () => {
+        function checkFeatures(featuresIds) {
+            cy.readStoreValue('state.features.selectedFeatures').then((features) => {
+                cy.wrap(features.length).should('be.equal', featuresIds.length)
+                features.forEach((feature) => {
+                    cy.wrap(featuresIds.includes(feature._id)).should('be.true')
+                })
+            })
+        }
+
+        beforeEach(() => {
+            // add intercept for layer config
+            cy.intercept('**/rest/services/all/MapServer/layersConfig**', {
+                fixture: 'layers.fixture',
+            })
+            // add intercept for the html popup
+            cy.intercept('**/MapServer/**/htmlPopup**', {
+                fixture: 'html-popup.fixture.html',
+            })
+        })
+
+        describe('Checks that the legacy bod layer id translate in the new implementation', () => {
+            function checkFeatures(featuresIds) {
+                cy.readStoreValue('state.features.selectedFeatures').then((features) => {
+                    cy.wrap(features.length).should('be.equal', featuresIds.length)
+
+                    const modifiedFeaturesIds = featuresIds.map(
+                        // feature.id returns a string in the form of `layer.id-feature.id`
+                        // thus a small adaptation to check we get the correct result
+                        (featureId) => `${features[0].layer.id}-${featureId}`
+                    )
+                    console.log(modifiedFeaturesIds)
+                    features.forEach((feature) => {
+                        cy.wrap(modifiedFeaturesIds.includes(feature.id)).should('be.true')
+                    })
+                })
+            }
+
+            beforeEach(() => {
+                // add intercept for layer config
+                cy.intercept('**/rest/services/all/MapServer/layersConfig**', {
+                    fixture: 'layers.fixture',
+                })
+                // add intercept for the html popup
+                cy.intercept('**/MapServer/**/htmlPopup**', {
+                    fixture: 'html-popup.fixture.html',
+                })
```

Comment:
I bet these two are already part of the default interception suite deployed when calling `.goToMapView`, and if it is not the case, please add them there instead (and remove them from your beforeEach context)